Makefile

```diff
@@ -2,6 +2,12 @@
 
 IMAGE_NAME ?= codeclimate/codeclimate-rubocop
 
+docs:
```

Comment:
I think...

1. this should depend on image (`docs: image`)
1. this should be added to the `PHONY` list at the top of the file
1. this shouldn't be the first task -- by convention, we always have `image` be the first task, so `make` will default to `make image`.